gatsby-config.js

```diff
@@ -45,63 +50,63 @@ const gatsbyConfig = {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'learn',
-        path: path.resolve('./content/learn'),
+        path: path.resolve(__dirname, './content/learn'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'sites',
-        path: path.resolve('./src/pages'),
+        path: path.resolve(__dirname, './src/pages'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'homepage',
-        path: path.resolve('./content/homepage'),
+        path: path.resolve(__dirname, './content/homepage'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'community',
-        path: path.resolve('./content/community'),
+        path: path.resolve(__dirname, './content/community'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'blog',
-        path: path.resolve('./content/blog'),
+        path: path.resolve(__dirname, './content/blog'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'data',
-        path: path.resolve('./src/data'),
+        path: path.resolve(__dirname, './src/data'),
       },
     },
     {
       resolve: 'gatsby-source-filesystem',
       options: {
         name: 'about',
-        path: path.resolve('./content/about'),
+        path: path.resolve(__dirname, './content/about'),
```

Comment:
I switched to prepending `__dirname` to prevent path manipulation as much as possible and allow relative sub paths.